setup.py

```diff
@@ -1,44 +1,81 @@
 #!/usr/bin/env python
 import os
-import re
 import pathlib
+import re
 
 from setuptools import setup
 
 
-def load_requirements(*requirements_paths):
```

Comment:
taken from https://github.com/openedx/xblock-sdk/blob/master/setup.py. everything below this method is just linter.